app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -9,6 +9,8 @@
 
   meta_tags["govuk:format"] = content_item_hash[:document_type] if content_item_hash[:document_type]
   meta_tags["govuk:schema-name"] = content_item_hash[:schema_name] if content_item_hash[:schema_name]
+  meta_tags["govuk:publishing-state"] = details_hash[:publishing_state] if details_hash[:publishing_state]
+  meta_tags["govuk:current-language"] = content_item_hash[:locale] if content_item_hash[:locale]
```

Comment:
Sorry, I pointed you at the wrong element. The HTML lang is usually `en` as parts of the page are always in English. However the `main` tag gets a lang attribute.

We set the lang on the `main` tag in government-frontend here:
https://github.com/alphagov/government-frontend/blob/master/app/views/layouts/application.html.erb#L25

Example:
https://www.gov.uk/government/news/foreign-secretary-statement-on-the-death-of-nelson-mandela.ru